pkgs/applications/science/misc/root/default.nix

```diff
@@ -193,8 +171,14 @@ stdenv.mkDerivation rec {
     # fatal error: module map file '/nix/store/<hash>-Libsystem-osx-10.12.6/include/module.modulemap' not found
     # fatal error: could not build module '_Builtin_intrinsics'
     "-Druntime_cxxmodules=OFF"
+
+    # clad fails to build on Darwin
+    "-Dclad=OFF"
```

Comment:
Probably use a single definition, if possible, like `-
Dclad=${if stdenv.isDarwin then "OFF" else "ON"}`

Reply:
Thanks for the suggestion!